src/static/particles.js

```diff
@@ -0,0 +1,81 @@
+import { tsParticles } from 'https://cdn.jsdelivr.net/npm/@tsparticles/engine@3.3.0/+esm';
+import { loadPolygonMaskPlugin } from 'https://cdn.jsdelivr.net/npm/@tsparticles/plugin-polygon-mask@3.3.0/+esm';
+import { loadFull } from 'https://cdn.jsdelivr.net/npm/tsparticles@3.3.0/+esm';
+
+let first_time = true;
+const field = document.getElementById('field');
+field.addEventListener('keypress', showResult);
+
+async function displayBackground() {
+    let new_config = await (await fetch("static/background.json")).json()
+
+    await tsParticles
+    .load({
+      id: "tsparticles2",
+      options: new_config,
+    });
+}
+
+displayBackground();
+
+async function displayTitle() {
+    const config = await (await fetch("static/title.json")).json();
+
+    await loadFull(tsParticles);
+    await loadPolygonMaskPlugin(tsParticles)
+
+    await tsParticles
+    .load({
+      id: "tsparticles",
+      options: config,
+    });
+}
+
+displayTitle();
+
+
+async function resultSlide(uid) {
+    let viteResult = document.getElementById('vite_result');
+
+    function resetTransform() {
+        if (!first_time) {
+            this.style.transform = "translateY(0)";
+        }
+    }
```

Comment:
Even if `resetTransform` is not used elsewhere, it's still better to declare it outside of the `resultSlide` 
 function